h2o-py/h2o/h2o.py

```diff
@@ -795,6 +797,7 @@ def parse_setup(raw_frames, destination_frame=None, header=0, separator=None, co
     :param partition_by: A list of columns the dataset has been partitioned by. None by default.
     :param quotechar: A hint for the parser which character to expect as quoting character. Only single quote, double quote or None (default) are allowed. None means automatic detection.
     :param escapechar: (Optional) One ASCII character used to escape other characters.
+    :param tz_adjustment: (Optional) Adjust the imported data timezone from GMT to cluster timezone.
 
```

Comment:
tz_adjustment is only for parsing parquet files, right?  Please add the clarification.